Gutenberg/General.md

```diff
@@ -6,32 +6,6 @@ All development details can be found on:
 - [Github](https://github.com/WordPress/gutenberg/)
 - [Handbook](https://developer.wordpress.org/block-editor/)
 
```

Comment:
I know that's in the docs, but I don't see the value of removing this from the handbook. Handbook will just give a high level, a quick overview of why we do things as we do, which is IMO better for new developers then diving into the technical docs right away.

Reply:
I would like to avoid on having the same thing on two places that is why I have removed it from the handbook.

Reply:
I'd leave this in. Handbook is something that new people read first, so it's a good general intro to what they'll be doing.

Reply:
I agree with Gabriel. Since this is the first thing new developers read, it will bring some high-level overview of our work.

Eightshift docs are more in-depth and dev oriented IMO.